src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -130,6 +130,44 @@ static Object* arraySpeciesCreate(ExecutionState& state, Object* originalArray,
     return Object::construct(state, C, 1, argv);
 }
 
+// http://ecma-international.org/ecma-262/10.0/#sec-flattenintoarray
+// FlattenIntoArray(target, source, sourceLen, start, depth [ , mapperFunction, thisArg ])
+static size_t flattenIntoArray(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
+    Value source = argv[0];
+    int64_t sourceLen = argv[1].toInteger(state);
+    size_t targetIndex = argv[2].toInteger(state);
```

Comment:
in 32bit, toInterger maximum value of toInteger is bigger than size_t.
How about to use int64_t?